File: Lua/Lua/customer.lua
-- Module with Cusotmer class that returns Customer table
Customer = {}
Customer.__index = Customer

function Customer.new(name, email, phone)
    local self = setmetatable({}, Customer)
    self.name = name
    self.email = email
    self.phone = phone
    self.purchases = {}
    return self
end

return Customer

File: rb/customer.rb
class Customer
  attr_accessor :name, :email, :phone, :purchases

  def initialize(name, email, phone)
    @name = name
    @email = email
    @phone = phone
    @purchases = []
  end
end

class Purchase
  attr_accessor :item, :quantity, :cost

  def initialize(item, quantity, cost)
    @item = item
    @quantity = quantity
    @cost = cost
  end
end
